app/qml/main.qml

```diff
@@ -666,6 +666,16 @@ ApplicationWindow {
         z: zPanel + 1000 // the most top
     }
 
+    StorageLimitDialog {
+        id: storageLimitDialog
+        onOpenSubscriptionPlans: {
+          if (__merginApi.apiSupportsSubscriptions) {
+            storageLimitDialog.close()
```

Comment:
you need to close it all the time.
also if (__merginApi.apiSupportsSubscriptions) can be in manageSubscriptionPlan()?